lib/identity/hostdata/ec2.rb

```diff
@@ -8,7 +8,7 @@ class EC2
       #
       # @return [EC2]
       def self.load
-        response = http.get('/2016-09-02/dynamic/instance-identity/document')
+        response = http.get('/2016-09-02/dynamic/instance-identity/document', { 'X-aws-ec2-metadata-token' => v2_token })
```

Comment:
we can drop the braces here since it's the last argument:
```suggestion
        response = http.get('/2016-09-02/dynamic/instance-identity/document', 'X-aws-ec2-metadata-token' => v2_token)
```